collector/barman_cloud_backup.go

```diff
@@ -84,12 +87,17 @@ func (b *BarmanCloudBackup) Scrape(ch chan<- prometheus.Metric, logger log.Logge
 			continue
 		}
 
-		success, err := strconv.Atoi(record[2])
+		exit_code, err := strconv.Atoi(record[2])
 		if err != nil {
 			logInvalidTsv(record)
 			continue
 		}
 
+		success := 0;
```

Comment:
Just note that semicolons are not used like this in Go

Reply:
Yep, I am aware. Sorry I missed that with the context switching. I thought i ran gofmt before committing, which would have picked this up as well.